daml-lf/transaction-test-lib/src/main/scala/lf/transaction/test/TransactionBuilder.scala

```diff
@@ -49,6 +49,8 @@ final class TransactionBuilder(
     nodes(parentId) match {
       case _: TxExercise | _: TxRollback =>
         children += parentId -> (children(parentId) :+ nodeId)
+      case _: TxRollBack =>
+        children += parentId -> (children(parentId) :+ nodeId)
```

Comment:
Is this clause ever hit, given the above one?

Reply:
good point, I think this was a botched merge, fixed in 0667cc76a35204a2165b1395ab54d5c457376900